src/Mayaqua/Encrypt.c

```diff
@@ -3684,6 +3685,10 @@ void FreeOpenSSLThreadState()
 #if OPENSSL_VERSION_NUMBER < 0x10100000L
 	CRYPTO_cleanup_all_ex_data();
 	ERR_remove_thread_state(NULL);
+#else
+#ifndef LIBRESSL_VERSION_NUMBER

```

Comment:
The two lines can be combined into one: `#elif !defined(LIBRESSL_VERSION_NUMBER)`

Reply:
Thanks. I'd like to keep my code unchanged.